docs/testing/regression-checklist/test-suites/extenstions/blocks/contact-form.md

```diff
@@ -0,0 +1,7 @@
+#### Contact Form
+
+- [ ] Publish a post with default form
```

Comment:
Shouldn't this be Submit the form or something along the lines? Can it actually create a user-facing post?

Reply:
oh, it's about publishing a post with Contact Form block, not about submitting the form. But this is indeed good test case :)